_docs/development.md

```diff
@@ -25,23 +25,26 @@ You'll need the following to get started:
 ## Create temba user for PostgreSQL
 
 {% highlight bash %}
-$ createuser temba --pwprompt -d
+$ sudo apt-get install postgresql-client postgresql-server-dev-9.3 postgis*
+$ sudo -u postgres createuser temba --pwprompt -d
 Enter password for new role:
 Enter it again:
+$ sudo adduser temba
```

Comment:
right, but for those who have not worked with postgres before, they would need at least a link to the relevant docs
Cheers,
-carl



> On Jan 29, 2015, at 1:42 PM, Nic Pottier <notifications@github.com> wrote:
> 
> In _docs/development.md <https://github.com/rapidpro/rapidpro/pull/124#discussion_r23791263>:
> 
> >  Enter password for new role:
> >  Enter it again:
> > +$ sudo adduser temba
> You're creating a system user for temba here to make authentication easier for postgres, but I don't think that is actually required.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/rapidpro/rapidpro/pull/124/files#r23791263>.
>